changelog.txt

```diff
@@ -1,5 +1,7 @@
 0.81
  * Fixed bug where creating a Realm instance with a hashcode collision no longer returned the wrong Realm instance.
+ * Updated Realm Core to version 0.89.1
+   - Fixed bug in "index rebuilding" (would delete the wrong column, causing crash).
```

Comment:
I like that we write updates to core as it might effect if users want to upgrade, but from a realm-android point of view there hasn't been any bug, as we have not exposed the functionality yet, and they cannot be exposed to it? So writing we fixed a bug doesn't make any sense (from a realm-android point of view)